### PR TITLE
NLS: adjust the i18n implementation to accept the .desktop file.

### PR DESCRIPTION
**Pull request by 01micko**
This involved creating the labwc-tweaks-gtk.desktop.in file and adding it to po/POTFILES.in and also adjusting data/meson.build to conditionally build the labwc-tweaks-gtk.desktop file. po/it.po Italian translation is updated with desktop translations. re #17

### CODE REVIEW

File: po/labwc-tweaks-gtk.pot [hunk 1]
```diff
@@ -8,13 +8,13 @@ msgid ""
...
 "Report-Msgid-Bugs-To: https://github.com/labwc/labwc-tweaks-gtk/issues\n"
-"POT-Creation-Date: 2024-10-16 10:10+1000\n"
+"POT-Creation-Date: 2024-10-20 09:34+1000\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
 "Language: \n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Type: text/plain; charset=CHARSET\n"
```
Comment (01micko) [inline po/labwc-tweaks-gtk.pot:17]: should be UTF-8

File: data/labwc-tweaks-gtk.desktop.in [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Encoding=UTF-8
```
Comment (01micko) [inline data/labwc-tweaks-gtk.desktop.in:2]: Encoding key is deprecated

**Comment by johanmalm** [comment]
LGTM. Not tested.

**Comment by 01micko** [comment]
Safe to merge, tested extensively. Thanks.

**Pull request merged**